db/models/topicsModel.js

```diff
@@ -0,0 +1,9 @@
+
+const db = require('../connection');
+
+exports.selectTopics = (arg) => {
```

Comment:
Could we avoid including 'arg' in this instance as it's not doing anything within the function?